pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
     "safety-gymnasium >= 0.1.0",
-    "torch >= 1.10.0, < 2.0.0a0",
+    "torch >= 1.10.0",
```

Comment:
Need also to unpin dependencies in `requirements.txt` and `conda-recipe.yaml`. Search `sync with` in the repo.

Reply:
Get it!